src/qt/intro.h

```diff
@@ -69,10 +69,13 @@ private Q_SLOTS:
     QString pathToCheck;
     uint64_t m_blockchain_size;
     uint64_t m_chain_state_size;
+    bool m_prune_set{false};
+    uint64_t pruneTarget;
```

Comment:
Could be renamed: s/`pruneTarget`/`m_prune_target`/ ?

Reply:
ACK